doc/book/getting-started/skeleton.md

```diff
@@ -44,12 +44,63 @@ $ composer serve
 This starts up a web server on localhost port 8080; browse to
 http://localhost:8080/ to see if your application responds correctly!
 
+> ### Setting a timeout
+>
+> Composer commands time out after 300 seconds (5 minutes). On Linux-based
+> systems, the `php -S` command that `composer serve` spawns continues running
+> as a background process, but on other systems halts when the timeout occurs.
+>
+> If you want the server to live longer, you can use the
+> `COMPOSER_PROCESS_TIMEOUT` environment variable when executing `composer
+> serve` to extend the timeout. As an example, the following will extend it
+> to a full day:
+>
+> ```bash
+> $ COMPOSER_PROCESS_TIMEOUT=86400 composer serve
```

Comment:
it can be registered to `~/.composer/config.json` that can be set by run command

``` 
composer config -g process-timeout 86400
```
So, no need to re-define it again.